package/rpm-config-SUSE.spec

```diff
@@ -22,9 +22,9 @@ Version:        0
 Release:        0
 Summary:        SUSE specific RPM configuration files
 License:        GPL-2.0-or-later
-Group:          System/Packages
+Group:          Development/Tools/Building
 URL:            https://github.com/openSUSE/rpm-config-SUSE
-Source:         %{name}-%{version}.tar.gz
+Source:         %{name}-%{version}.tar.xz
```

Comment:
This is actually wrong even upstream, since this really should be a SourceURL pointing to upstream... 😢

Reply:
The tarball is generated from git using a service. So doesn't really apply anyway :-/

Reply:
actually since the default compression for src rpms is gzip anyway, we could just as well use tar directly, eg https://build.opensuse.org/request/show/886635